docs/source/index.rst

```diff
@@ -5,67 +5,62 @@
 
 .. role:: red
 
-Welcome to the Docs for Forest and pyQuil!
-==========================================
+Welcome to the Docs for the Forest SDK!
+=======================================
 
+The Rigetti Forest `Software Development Kit <http://rigetti.com/forest>`_ includes pyQuil, our Quil Compiler (quilc),
```

Comment:
...includes pyQuil, the Quil Compiler (`quilc`), and the Quantum Virtual Machine (`qvm`).